kyaml/yaml/fns.go

```diff
@@ -68,7 +72,7 @@ func (e ElementSetter) isMappingSetter() bool {
 	return e.Key != "" && e.Value != ""
 }
 
```

Comment:
Filter is an exportable function. Is that okay to change the function name? Will it break the some potential callers?

Reply:
oh, I see Filter not removed.